operators/src/engine/operator.rs

```diff
@@ -195,3 +195,13 @@ impl From<Box<InitializedRasterOperator>> for TypedInitializedOperator {
         TypedInitializedOperator::Raster(operator)
     }
 }
+
+#[macro_export]
+macro_rules! call_on_typed_operator {
```

Comment:
i thought we already have a macro like this

Reply:
Where?

Reply:
i don't know. i just thought i have seen one just like this